paasta_tools/docker_wrapper.py

```diff
@@ -33,7 +33,7 @@
 
 
 def parse_env_args(args):
-    result = {}
+    result = {k: v for k, v in os.environ.items()}
```

Comment:
This is just `dict(os.environ.items())` right?

Reply:
Oops, sorry, missed this comment.
Yes, it is just `dict(os.environ.items())`.
Unfortunately, I've already merged it.